### ISSUE TITLE
Stack information out of date?

### ISSUE THREAD

**Issue by FranklinChen** [ucsd-progsys/lh-plugin-demo#6]
`stack.yaml` and `stack.yaml.github` look old. In particular, I just got a Mac Studio and can't build with the current `stack.yaml`:

```
$ stack build
No setup information found for ghc-8.10.2 on your platform.
This probably means a GHC bindist has not yet been added for OS key 'macosx-aarch64'.
Supported versions: ghc-8.10.5, ghc-8.10.6, ghc-8.10.7, ghc-9.0.2, ghc-9.2.1, ghc-9.2.2
```

**Comment by ranjitjhala** [comment]
Interesting! Did you try changing the line

   compiler: ghc-8.10.2

to

   compiler: ghc-8.10.7

Does that work?

**Comment by FranklinChen** [comment]
It took forever to build, but yes, it did work!

**Comment by facundominguez** [comment]
The configuration has been updated in #14

**Comment by igormoreno** [comment]
How should I change the [stack.yaml](https://github.com/ucsd-progsys/lh-plugin-demo/blob/main/stack.yaml) file to use the new [9.0.2](https://github.com/ucsd-progsys/liquidhaskell/releases/tag/v9.0.2) release of Liquid Haskell?

**Comment by igormoreno** [comment]
wow! that was fast :-) thank you

I'm finding a couple of problems though:

- when I use the configuration in [stack.yaml](https://github.com/ucsd-progsys/lh-plugin-demo/blob/main/stack.yaml) I get:
```
           liquid-base needed, but the Stack
                       configuration has no
                       specified version (latest
                       matching version is
                       4.15.1.0)
```
I tried to add that package to the extra dependencies and then it complains about  `liquid-ghc-prim` and when I added that to the dependencies some compilation error happened

- when I use the configuration in [stack/stack-9.0.2.yaml](https://github.com/ucsd-progsys/lh-plugin-demo/blob/main/stack/stack-9.0.2.yaml) I get:
```
Error: [S-636]
Stack looks for packages in the directories configured in
the 'packages' and 'extra-deps' fields defined in your stack.yaml
The current entry points to /Users/igormoreno/Documents/academic/usi/phd/luce-researchlab/structured-notional-machines/,
but no .cabal or package.yaml file could be found there.
```
I solved that by deleting one of the dots in the third line

**Comment by facundominguez** [comment]
> solved that by deleting one of the dots in the third line

Yeah, that looks about right. The location of packages must be relative to the location of the stack.yaml file.

> when I use the configuration in stack.yaml I get:

Understanding this is going to require looking at the project files. But that `stack.yaml` is not meant to be used with the 9.0.2 release.

**Comment by igormoreno** [comment]
ok. Thank you.

Now with [stack/stack-9.0.2.yaml](https://github.com/ucsd-progsys/lh-plugin-demo/blob/main/stack/stack-9.0.2.yaml) I have stumbled over a problem that I found was already previously described:
- https://github.com/haskell/haskell-language-server/issues/3134
- https://github.com/commercialhaskell/stack/issues/5935

that seems to be somehow in the interplay between Liquid Haskell, the haskell-language-server, and Stack.

**Comment by facundominguez** [comment]
@igormoreno, @ranjitjhala, I'd recomend closing this issue as resolved and opening a new one for the HLS troubles if they might come from a misconfiguration in this repo.
